show_source/show_source.py

```diff
@@ -1,6 +1,9 @@
 import os
 import logging
-from urlparse import urljoin
+try:
+    from urllib.parse import urljoin
+except ImportError:  # Python 2 fallback:
+    from urlparse import urljoin
```

Comment:
Since it's pretty safe to assume that `pelican` is installed and `six` is a dependency of it:
```python
from six.moves.urllib.parse import urljoin
```

Reply:
Ok, fixed